### PR TITLE
Fix modified date used when comparing items

### PR DESCRIPTION
**Pull request by h2kuo**
For files, we should be using contentModifiedDate, but for folders and bookmarks, we should use modifiedDate.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @h2kuo has signed the CLA. Thanks for the pull request!

File: BoxBrowseSDK/BoxBrowseSDK/Categories/BOXItem+BOXBrowseSDKAdditions.m [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//  Copyright © 2016 BOX. All rights reserved.
+//
+
+#import "BOXItem+BOXBrowseSDKAdditions.h"
+
+@implementation BOXItem (BOXBrowseSDKAdditions)
+
+- (NSDate *)effectiveUpdateDate
+{
+    return self.isFile ? self.contentModifiedDate : self.modifiedDate;
```
Comment (thuyBox) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/BOXItem+BOXBrowseSDKAdditions.m]: I think you want to check self.isFolder ? self.modifiedDate : self.contentModifiedDate

**Comment by thuyBox** [comment]
+2

File: BoxBrowseSDK/BoxBrowseSDK/Categories/BOXItem+BOXBrowseSDKAdditions.m (hunk 1 above)
Comment (h2kuo) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/BOXItem+BOXBrowseSDKAdditions.m]: Discussed offline, want to use modifiedDate for bookmarks as well.

**Pull request merged**